paper/paper.md

```diff
@@ -9,12 +9,16 @@ tags:
 authors:
  - name: Christos Theodoropoulos
    orcid: 0000-0002-5395-0347
-   affiliation 1: Hellenic Centre for Marine Research
-   affiliation 2: National Technical University of Athens
+   affiliation: 1, 2
  - name: Nikolaos Skoulikidis
```

Comment:
I added the ORCID of Anastasios Stamou, however, the other author does not have one. If it is necessary, please let me know ok?